node/full_node_integration_test.go

```diff
@@ -264,22 +269,27 @@ func testSingleAggregatorTwoFullNode(t *testing.T, useBlockExchange bool) {
 	node3 := nodes[2]
 
 	require.NoError(node1.Start())
+	defer func() {
+		aggCancel()
```

Comment:
I'd move this to right after the creation of the context instead of putting the cancellation of the context as dependent on the node stopping defer statement. 

That'll be easier to maintain as well in terms of standards. Create then defer close.